src/components/tabs/PillTabs.js

```diff
@@ -0,0 +1,73 @@
+const React = require('react');
+
+const ButtonGroup = require('../ButtonGroup');
+const Tabbable = require('./Tabbable');
+
+const StyleConstants = require('../../constants/Style');
+
+const PillTabs = Tabbable(({
```

Comment:
I wonder if this pattern would be more approachable for some team members if the component was assigned here:

const Pill = (props) => {};

and then exported with Tabbable at the bottom.

module.exports = Tabbable(Pill);